chaos_genius/views/download_view.py

```diff
@@ -0,0 +1,215 @@
+import csv
+import io
+
+from flask import Blueprint, Response, jsonify, request
+
+from chaos_genius.controllers.kpi_controller import get_kpi_data_from_id
+from chaos_genius.core.rca.rca_utils.api_utils import (
+    kpi_line_data,
+    rca_analysis,
+    rca_hierarchical_data,
+)
+from chaos_genius.views.anomaly_data_view import get_overall_data_points
+
+blueprint = Blueprint("downloads", __name__)
+
+
+def iter_csv(data):
```

Comment:
Would be better to have this in `chaos_genius.utils.utils` or make it a private function in this module